src/api/api-add-categories.ts

```diff
@@ -1,14 +1,14 @@
 import axios from "axios";
 
-export interface categoriesObjModel {
+export interface CategoriesObjModel {
   parentCategoryId: string;
   name: string;
   description: string;
   icon: string;
   imagePath: string;
 }
 
-export interface categoriesObjResponse {
+export interface CategoriesObjResponse {
```

Comment:
Ok, your response model contains subcategories but you category modes not. Why is that? I mean what is the difference?

Reply:
it's my mistake, I forgot to delete it